miniscope_io/__init__.py

```diff
@@ -8,6 +8,7 @@
 DATA_DIR = BASE_DIR / 'data'
 CONFIG_DIR = DATA_DIR / 'config'
 DEVICE_DIR = BASE_DIR / 'devices'
+TESTDATA_DIR = BASE_DIR.parent / 'tests' / 'data'
```

Comment:
we'll want this to be in `tests/conftest.py` - this directory won't be present in the wheel or sdist of the package, and is only relevant at test time :)